minio-client/deployment/src/main/java/io/quarkiverse/minio/client/deployment/devservices/DevServicesMinioProcessor.java

```diff
@@ -312,13 +315,14 @@ private static final class MinioContainer extends GenericContainer<MinioContaine
         private String hostName = null;
 
         private MinioContainer(DockerImageName dockerImageName, int fixedExposedPort, String accessKey, String secretKey,
-                boolean useSharedNetwork) {
+                Map<String, String> containerEnv, boolean useSharedNetwork) {
             super(dockerImageName);
             this.port = fixedExposedPort;
             this.useSharedNetwork = useSharedNetwork;
             withExposedPorts(DEVSERVICE_MINIO_PORT, DEVSERVICE_MINIO_CONSOLE_PORT);
             withEnv("MINIO_ACCESS_KEY", accessKey);
             withEnv("MINIO_SECRET_KEY", secretKey);
+            withEnv(containerEnv);
```

Comment:
If `env` is null, the `withEnv` will throw a `NullPointerException`

Reply:
Yes. I verified it as well, it is never `null`.